resources/js/components/apis/CreateAddress.vue

```diff
@@ -0,0 +1,144 @@
+                    id="email"
+                    ref="email"
+                    @input="updateUserAddress"
+                />
+            </div>
+        </div>
+        <div class="row">
+            <div class="col-6 form-group">
+                <label for="area">Area</label>
+                <input
+                    type="text"
+                    class="form-control"
+                    placeholder="Area"
+                    name="area"
+                    id="area"
+                    ref="area"
+                    @input="updateUserAddress"
+                />
+            </div>
+            <div class="col-6 form-group">
+                <label for="street">Street</label>
+                <input
+                    type="text"
+                    class="form-control"
+                    placeholder="Street"
+                    name="street"
+                    id="street"
+                    ref="street"
+                    @input="updateUserAddress"
+                />
+            </div>
+        </div>
+        <div class="row">
+            <div class="col-4 form-group">
+                <label for="building">Building</label>
+                <input
+                    class="form-control"
+                    type="text"
+                    placeholder="Building"
+                    name="building"
+                    id="building"
+                    ref="building"
+                    @input="updateUserAddress"
+                />
+            </div>
+            <div class="col-4 form-group">
+                <label for="floor">Floor</label>
+                <input
+                    type="text"
+                    class="form-control"
+                    placeholder="Floor"
+                    name="floor"
+                    id="floor"
+                    ref="floor"
+                    @input="updateUserAddress"
+                />
+            </div>
+            <div class="col-4 form-group">
+                <label for="apt">Apartment</label>
+                <input
+                    type="text"
+                    class="form-control"
+                    placeholder="Apartment Number"
+                    name="apt"
+                    id="apt"
+                    ref="apt"
+                    @input="updateUserAddress"
+                />
+            </div>
+            <button
+                ref="submit_button"
+                class="btn btn-primary"
+                v-on:click="createAddress"
+                disabled
+            >
+                Create Address
+            </button>
+        </div>
+    </div>
+</template>
+
+<script>
+export default {
+    data() {
+        return {
+            regexp: /^(([^<>()\[\]\\.,;:\s@"]+(\.[^<>()\[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,24}))$/,
+        };
+    },
+    computed: {
+        validate() {
+            this.$refs["email"].style =
```

Comment:
utilize the class binding feature of vue instead of querying the dom and writing style